src/main/scala/com/microsoft/sqlserver/jdbc/spark/SQLServerBulkJdbcOptions.scala

```diff
@@ -1,24 +1,31 @@
 package com.microsoft.sqlserver.jdbc.spark
 
 import java.sql.Connection
+
 import org.apache.spark.sql.catalyst.util.CaseInsensitiveMap
-import org.apache.spark.sql.execution.datasources.jdbc.JdbcOptionsInWrite
+import org.apache.spark.sql.execution.datasources.jdbc.{JDBCOptions, JdbcOptionsInWrite}
 
 class SQLServerBulkJdbcOptions(val params: CaseInsensitiveMap[String]) extends JdbcOptionsInWrite(params) {
 
     def this(params: Map[String, String]) = this(CaseInsensitiveMap(params))
-    
+
     // Save original parameters for when a JdbcBulkOptions instance is passed
     // from the Spark driver to an executor, which loses the reference to the
     // params input in memory
     override val parameters = params
 
+    val dbtable = params.getOrElse("dbtable", null)
+    val databaseName = params.getOrElse("databaseName", null)
+
     val user = params.getOrElse("user", null)
     val password = params.getOrElse("password", null)
-    val dbtable = params.getOrElse("dbtable", null)
 
+  //  AAD Authentication
```

Comment:
Nit, fix indent.

Reply:
This has been adjusted with all indentations corrected in the separate PR.

Reply:
Thanks for the broader PR. Request this quick fix here as well.

Reply:
Updated.